### DISCUSSION TITLE
如何使 Input 的 addonAfter 整个区域可以点击，而不是单单图标部分

### DISCUSSION THREAD

**Discussion by hihuangwei** [ant-design/ant-design#37897] [General]
### What problem does this feature solve?

如何使 Input 的 addonAfter 整个区域可以点击，而不是单单图标部分

![微信截图_20221008204818](https://user-images.githubusercontent.com/3166799/194708452-9012832d-fa3e-4053-8551-ad3e9bda9008.png)

**Comment by wd4752** [comment]
同问，有解决了吗
